docs/status.md

```diff
@@ -20,126 +20,127 @@ one of the following tracking labels.
     - **yes**: complete
     - **wip**: semi-complete: Work in progress or under review.
     - **no**: not complete yet, but part of [the roadmap](https://github.com/openxla/stablehlo#roadmap).
- - Customized labels
-    - Verifier
-       - **match-xla**:  verifier in sync with  [XLA semantics](https://www.tensorflow.org/xla/operation_semantics).
-       - **match-spec**: verifier in sync with [StableHLO semantics](https://github.com/openxla/stablehlo/blob/main/docs/spec_draft.md).
-
+ - Customized labels for Verifier and Type Inference 
+    - **yes(match-xla)**:  in sync with  [XLA semantics](https://www.tensorflow.org/xla/operation_semantics).
```

Comment:
Agreed that the current form is pretty verbose. I'd propose a tweak to Kevin's proposal:
  * Change `yes(match-spec)` to `yes`, given that our ultimate goal is to match the spec, and matching XLA is only an intermediate step in that direction.
  * Change `yes(match-xla)` to just `yes*` and explain what it means in the section that talks about customized labels. This will make the table much easier on the eyes, similarly to what Kevin proposes.
  * `yes(need-revisit)` sounds good.
  * Change `no-plan` to `infeasible`. "No plan" can be interpreted in multiple ways, including "it's feasible to implement but we won't".

Reply:
Sounds great, addressed.